packages/node_modules/@webex/webex-core/src/lib/services/index.js

```diff
@@ -4,14 +4,18 @@
 import {registerInternalPlugin} from '../../webex-core';
```

Comment:
I amended the server-error interceptor to the services plugin, where it can safely mark urls as failed.

Reply:
Seems logical